### DISCUSSION TITLE
multiple terrain file

### DISCUSSION THREAD

**Discussion by ankitpentashield** [sogelink-research/ctod#14] [Q&A]
![image](https://github.com/user-attachments/assets/bef4e24b-45fe-4e33-8943-4ba6096bb0d1)
I have high res dem file for 3 operation site of non overlapping are how can I load and visualize all at the same time.

**Comment by tebben** [comment]
You can create a `.vrt` file and point CTOD to the vrt file instead of a tif.

```sh
gdalbuildvrt india.vrt Srinagar_DEM_30m.tif delhi_dem.tif mosaic_almora_dem_float32.tif
```

**Comment by ankitpentashield** [comment]


**Reply by tebben** [reply]
I'm not a LLM where you can just throw images into

**Reply by ankitpentashield** [reply]
somehow cotd is not able to find cogt_india/merged.vrt although it is present in the same folder as test_cog.tif
